ibm_cloud_sdk_core/service.py

```diff
@@ -0,0 +1,441 @@
+# coding: utf-8
+
+# Copyright 2019 IBM All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import dateutil.parser as date_parser
+import os
+from os.path import dirname, isfile, join, expanduser, abspath
+import platform
+import json as json_import
+import sys
+import requests
+import warnings
+from requests.structures import CaseInsensitiveDict
+from .version import __version__
+from .utils import has_bad_first_or_last_char, remove_null_values, cleanup_values
+from .iam_token_manager import IAMTokenManager
+from .detailed_response import DetailedResponse
+from .api_exception import ApiException
+
+try:
+    from http.cookiejar import CookieJar  # Python 3
+except ImportError:
+    from cookielib import CookieJar  # Python 2
+
+# Uncomment this to enable http debugging
+# try:
+#    import http.client as http_client
+# except ImportError:
+#    # Python 2
+#    import httplib as http_client
+# http_client.HTTPConnection.debuglevel = 1
+
+class Service(object):
+    BEARER = 'Bearer'
+    X_WATSON_AUTHORIZATION_TOKEN = 'X-Watson-Authorization-Token'
```

Comment:
I think Logan has removed a similar definition from the Java core because it was deprecated.   Should this also be removed from the python core???
I think, in general, we want to remove Watson-related references from the core packages... at least where reasonable.